OpenRA.Mods.Common/Util.cs

```diff
@@ -11,16 +11,18 @@
 
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
 using OpenRA.GameRules;
-using OpenRA.Graphics;
 using OpenRA.Mods.Common.Traits;
 using OpenRA.Primitives;
 using OpenRA.Support;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common
 {
+	public enum InaccuracyType { Maximum, PerCellIncrement, Absolute }
```

Comment:
Does this really belong to the util file? :D

Reply:
Same answer as below.